web/l10n/en.json

```diff
@@ -269,6 +272,7 @@
     "eventIgnitionOff": "Ignition is OFF",
     "eventMaintenance": "Maintenance required",
     "eventTextMessage": "Text message received",
+    "eventDriverUnauthorized": "Driver Unauthorized",
```

Comment:
Do we need this event? I think instead we should have an event when a new driver has authorised. I believe in most cases this feature will be used for informational purposes only, not to detect unauthorised access.